anthos-bm-gcp-terraform/test/unit/main_test.go

```diff
@@ -197,6 +198,8 @@ func TestUnit_MainScript(goTester *testing.T) {
 			ingressLBModule = append(ingressLBModule, childModule)
 		} else if strings.HasSuffix(moduleAddress, "configure_controlplane_lb[0]") {
 			controlplaneLBModule = append(controlplaneLBModule, childModule)
+		} else if strings.HasSuffix(moduleAddress, "gke_hub_membership") {
+      gkeHubMembership = append(gkeHubMembership, childModule)
```

Comment:
Indent to match the other lines

Reply:
I see what happened: I used spaces but the file uses tabs.